From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chun-Hung Wu <chun-hung.wu@mediatek.com>
Date: Tue, 25 Aug 2020 15:48:16 +0800
Subject: FROMLIST: clk: Export clk_register_composite

clk_register_composite() will be used in mediatek's
clock kernel module, so export it to GPL modules.

Bug: 154743265
Link: https://lore.kernel.org/lkml/1595493126-21611-1-git-send-email-wendell.lin@mediatek.com/
Change-Id: I658a5073b9d6b33ff898f87133c5fbeb693ac6f9
Signed-off-by: Wendell Lin <wendell.lin@mediatek.com>
Signed-off-by: Chun-Hung Wu <chun-hung.wu@mediatek.com>
(cherry picked from commit a254a08c1327bfd8c35b7d1d904d015ca3ab674f)
---
 drivers/clk/clk-composite.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/clk-composite.c b/drivers/clk/clk-composite.c
--- a/drivers/clk/clk-composite.c
+++ b/drivers/clk/clk-composite.c
@@ -360,6 +360,7 @@ struct clk *clk_register_composite(struct device *dev, const char *name,
 		return ERR_CAST(hw);
 	return hw->clk;
 }
+EXPORT_SYMBOL_GPL(clk_register_composite);
 
 struct clk *clk_register_composite_pdata(struct device *dev, const char *name,
 			const struct clk_parent_data *parent_data,
